index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+    <header class="toolbar">
+        <a href="#" ><p>My logo </p></a>
+        <div class ="vector">
+            <a href="#" >   <img src = "./Icons/Union.png"> 
+            </a>
+        </div>
+          
+    </header>
+      <div class="background">
+       <section class="headline">
+        <div class="header"><h1>I’m Mariane Glad to see you!</h1></div>
+        <div class="info"><h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3></div>
```

Comment:
- [ ] As requested by the previous reviewer please use the same font family. Please see the content and fix it.
- [ ] Also use the same color and font for `let connect` as well.
### Your design
![your](https://user-images.githubusercontent.com/81727988/219424934-79963db2-cedd-43ea-8088-fc73097bf1f3.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/219424969-55d9eed3-cce7-45d8-bb43-5fdf7bff335e.png)